webpack/assets/javascripts/foreman_store.js

```diff
@@ -0,0 +1,34 @@
+import { get } from 'lodash';
+import store from './react_app/redux';
+
+/**
+ * Observe a state from the store for changes
+ * @param  {String}   state a sub object of the store
+ * @param  {Function} onChange  a callback to run when the store has been changed
+ * @return {Function}           unsubscribe function
+ */
+export function observeStore(state, onChange) {
```

Comment:
1. IMO it should be called `subscribe`.
  It will be used as `tfm.store.subscribe(...)`.
2. You can use new js syntax `const subscribe = () => { ... }`
3. `state` can be a bit confusing name for this variable, could you name it `stateLocation`?

Reply:
subscribe is the low level API which redux doesn't recommend to use directly, so I think it would be better to use `observeStore`
I took this implementation from redux suggestion:
https://github.com/reduxjs/redux/issues/303#issuecomment-125184409